packages/expo-firebase-analytics/android/src/main/java/expo/modules/firebase/analytics/FirebaseAnalyticsModule.java

```diff
@@ -0,0 +1,193 @@
+package expo.modules.firebase.analytics;
+
+import android.app.Activity;
+import android.content.Context;
+
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.analytics.FirebaseAnalytics;
+
+import org.unimodules.core.ExportedModule;
+import org.unimodules.core.ModuleRegistry;
+import org.unimodules.core.Promise;
+import org.unimodules.core.arguments.MapArguments;
+import org.unimodules.core.interfaces.ActivityProvider;
+import org.unimodules.core.interfaces.ExpoMethod;
+import org.unimodules.core.interfaces.RegistryLifecycleListener;
+
+import java.util.Map;
+
+import androidx.annotation.Nullable;
+
+public class FirebaseAnalyticsModule extends ExportedModule implements RegistryLifecycleListener {
+    private static final String NAME = "ExpoFirebaseAnalytics";
+
+    private Activity mActivity;
+
+    public FirebaseAnalyticsModule(Context context) {
+        super(context);
+    }
+
+    @Override
+    public String getName() {
+        return NAME;
+    }
+
+    @Override
+    public void onCreate(ModuleRegistry moduleRegistry) {
+        ActivityProvider mActivityProvider = moduleRegistry.getModule(ActivityProvider.class);
+        mActivity = mActivityProvider.getCurrentActivity();
+    }
+
+    @Override
+    public Map<String, Object> getConstants() {
+        final Map<String, Object> constants = new HashMap<>();
```

Comment:
Formatting settings of Java in `expo` are set to 2 lines indentation — could you check that Android Studio respects that and reformat the file? (Android Studio has a tool for that)